docs/cli.md

```diff
@@ -0,0 +1,687 @@
+  ```
+
+  </td>
+  <td>
+    <img src="images/cli003.png" class="invert-colors-in-dark-mode" alt="ls example" width="470px" />
+  </td>
+  </tr>
+</table>
+
+### `pwd`
+`pwd` prints the path of the present working directory.
+
+<table>
+  <tr>
+  <td markdown="1">
+
+  ```console
+  $ pwd
+  /Users/ohjun/Desktop/project
+  ```
+
+  </td>
+  <td>
+    <img src="images/cli004.png" class="invert-colors-in-dark-mode" alt="pwd example" width="470px" />
+  </td>
+  </tr>
+</table>
+
+
+### `mkdir`
+`mkdir` creates a directory.
+
+<table>
+  <tr>
+  <td markdown="1">
+
+  ```console
+  $ mkdir myfolder
+  ```
+
+  </td>
+  <td>
+    <img src="images/cli012.png" class="invert-colors-in-dark-mode" alt="mkdir example" width="470px"/>
+  </td>
+  </tr>
+</table>
+
+### `touch`
+`touch` creates an empty file.
+
+<table>
+  <tr>
+  <td markdown="1">
+
+  ```console
+  $ ls
+  example.txt main.cpp stuff stuff2
+  $ touch euchre.cpp
+  $ ls
+  example.txt main.cpp stuff stuff2 euchre.cpp 
+  ```
+
+  </td>
+  <td>
+    <img src="images/cli006.png" class="invert-colors-in-dark-mode" alt="touch example" width="470px"/>
+  </td>
+  </tr>
+</table>
+
+### `rm`
+`rm` removes (deletes) a file. `rm -rf` removes a directory.
+
+<table>
+  <tr>
+  <td markdown="1">
+
+  ```console
+  $ rm euchre.cpp
+  $ rm -r stuff2/
+  ```
+
+  </td>
+  <td>
+    <img src="images/cli013.png" class="invert-colors-in-dark-mode" alt="rm example" width="470px" />
+  </td>
+  </tr>
+</table>
+
+### `cd`
+`cd` changes directory.
+
+<table>
+  <tr>
+  <td markdown="1">
+
+  ```console
+  $ cd stuff/
+  ```
+
+  </td>
+  <td>
+    <img src="images/cli014.png" class="invert-colors-in-dark-mode" alt="cd example" width="470px"/>
+  </td>
+  </tr>
+</table>
+
+### `mv`
+`mv` moves a file or directory into a different directory.
+
+<table>
+  <tr>
+  <td markdown="1">
+
+  ```console
+  $ mv main.cpp stuff/
+  ```
+
+  </td>
+  <td>
+    <img src="images/cli005.png" class="invert-colors-in-dark-mode" alt="mv example" width="470px"/>
+  </td>
+  </tr>
+</table>
+
+`mv` is also used to rename a file or directory.
+
+<table>
+  <tr>
+  <td markdown="1">
+
+  ```console
+  $ mv example.txt new_name.txt
+  ```
+
+  </td>
+  <td>
+    <img src="images/cli015.png" class="invert-colors-in-dark-mode" alt="mv rename example" width="470px"/>
+  </td>
+  </tr>
+</table>
+
+### `cp`
+`cp` copies a file.
+
+<table>
+  <tr>
+  <td markdown="1">
+
+  ```console
+  $ cp new_name.txt stuff/
+  ```
+
+  </td>
+  <td>
+    <img src="images/cli016.png" class="invert-colors-in-dark-mode" alt="cp example"/>
+  </td>
+  </tr>
+</table>
+
+### `open` / `wslview`
+On macOS, `open` opens a file or directory with the default application ([docs](https://ss64.com/osx/open.html)).
+
+On WSL (Windows), `wslview` opens a file or directory with the default application ([docs](https://wslutiliti.es/wslu/man/wslview.html)).
+
+<video controls autoplay loop style="width: 100%; max-width: 640px; max-height: 480px;">
```

Comment:
I changed all the videos to mp4 and add the loop attribute.  I'm having trouble with autoplay but it might just be my browser.  Give it a try.

Reply:
Autoplay seems to work on Firefox and Safari, but not on Chrome on my end.

Reply:
Well, I guess this is the best we can do then.